Tests/DataCollector/DoctrineDataCollectorTest.php

```diff
@@ -49,6 +49,32 @@ public function testCollectEntities()
         $this->assertCount(2, $entities['default']);
     }
 
+    public function testDoNotCollectEntities()
```

Comment:
```suggestion
    public function testDoNotCollectEntities() : void
```

Reply:
I tend to follow the lead of the current file, and no other method as return type.